File: external/webroot/md/codeExample/Face-API-Verify/FaceVerify.cs
using System;
using System.IO;
using System.Net;
using System.Text;
using System.Web;

namespace com.baidu.ai
{
	public class Verify
	{
		// 人脸认证
		public static String verify()
		{
			string token = "########"
			string base645 = FileUtils.getFileBase64("/work/ai/images/face/search5.jpg");
			string host = "https://aip.baidubce.com/rest/2.0/face/v2/verify?access_token=" + token;
			Encoding encoding = Encoding.Default;
			HttpWebRequest request = (HttpWebRequest)WebRequest.Create(host);
			request.Method = "post";
			request.ContentType = "application/x-www-form-urlencoded";
			request.KeepAlive = true;
			String str = "images=" + HttpUtility.UrlEncode(base645);
			str += "&uid=songqingyun&top_num=1";
			byte[] buffer = encoding.GetBytes(str);
			request.ContentLength = buffer.Length;
			request.GetRequestStream().Write(buffer, 0, buffer.Length);
			HttpWebResponse response = (HttpWebResponse)request.GetResponse();
			StreamReader reader = new StreamReader(response.GetResponseStream(), Encoding.Default);
			string result = reader.ReadToEnd();
			Console.WriteLine("人脸认证:");
			Console.WriteLine(result);
			return result;
		}
	}
}

File: external/webroot/md/codeExample/Face-API-Verify/FaceVerify.py
# encoding:utf-8
import base64
import urllib
import urllib2

from token import access_token

'''
人脸查找之认证接口
'''

verifyUrl = "https://aip.baidubce.com/rest/2.0/face/v2/verify"
# 参数uid：用户id（由数字、字母、下划线组成）images：图像base64编码,多张图片半角逗号分隔
params = {"uid": "testuid",
          "images": "【图片base64编码】"}
params = urllib.urlencode(params)
access_token = access_token.AuthService()
verifyUrl = verifyUrl + "?access_token=" + access_token
request = urllib2.Request(url=verifyUrl, data=params)
request.add_header('Content-Type', 'application/x-www-form-urlencoded')
response = urllib2.urlopen(request)
content = response.read()
if content:
    print content
